src/ms-review.tex

```diff
@@ -230,30 +230,30 @@ \section{Introduction}
 
 
 %% \gammaray astronomy
-In modern astronomy astrophysical objects are studied across
+Modern astronomy is defined by the possibility to observe and study astrophysical sources across
```

Comment:
I find the "defined by" a bit strong, like there was only one valid definition. Suggestion below:

```suggestion
Modern astronomy offers the possibility to observe and study astrophysical sources across
```